README.rdoc

```diff
@@ -170,7 +170,7 @@ This would make all instance variables inside the helpers safe and intuitive. I'
 
 == License
 
-Copyright (c) 2009-2012, Brad Gessler
+Copyright (c) 2009-2019, Brad Gessler
```

Comment:
I like that you're being a good citizen by updating these dates, but last I checked they don't actually do anything. How about we simplify them by removing the dates? I think it should also be ©️ Poll Everywhere now, not just Brad.

Reply:
Thanks Mike; I fixed the copyright holder. I thought the date was used to indicate any year in which a change was made to the software covered by the copyright? I think, in that case, I should have used `2009-2012, 2019`. I also didn't change anything substantive, so I set the date back to 2012.

Reply:
My suggestion was to remove the dates entirely, because they don't perform any legal function and they look bad when they get out of date.